app/router.js

```diff
@@ -22,12 +22,13 @@ const Router = Ember.Router.extend({
 });
 
 Router.map(function() {
-    this.route('index', {path: 'preprints'});
-    this.route('submit', {path: 'preprints/submit'});
-    this.route('discover', {path: 'preprints/discover'});
-    this.route('content', { path: '/:preprint_id' });
-    this.route('page-not-found', { path: 'preprints/*bad_url'});
-    this.route('page-not-found', {path: 'preprints/page-not-found'});
+  this.route('index', {path: 'preprints'});
```

Comment:
indentation change is causing builds to fail https://travis-ci.org/CenterForOpenScience/ember-preprints/builds/172151371#L333